ostelco-core/Models/StageDecider.swift

```diff
@@ -38,11 +37,10 @@ public class LocalContext {
     public var serverIsUnreachable: Bool
     public var hasCameraProblem: Bool
     
-    public init(hasFirebaseToken: Bool = false, hasAgreedToTerms: Bool = false, hasSeenNotificationPermissions: Bool = false, hasSeenVerifyIdentifyOnboarding: Bool = false, selectedVerificationOption: IdentityVerificationOption? = nil, myInfoCode: String? = nil, hasSeenESimOnboarding: Bool = false, hasSeenESIMInstructions: Bool = false, hasSeenAwesome: Bool = false, hasCompletedJumio: Bool = false, hasCompletedAddress: Bool = false, serverIsUnreachable: Bool = false, locationProblem: LocationProblem? = nil, hasSeenRegionOnboarding: Bool = false, hasSeenLocationPermissions: Bool = false, hasCameraProblem: Bool = false) {
+    public init(hasFirebaseToken: Bool = false, hasAgreedToTerms: Bool = false, hasSeenNotificationPermissions: Bool = false, selectedVerificationOption: IdentityVerificationOption? = nil, myInfoCode: String? = nil, hasSeenESimOnboarding: Bool = false, hasSeenESIMInstructions: Bool = false, hasSeenAwesome: Bool = false, hasCompletedJumio: Bool = false, hasCompletedAddress: Bool = false, serverIsUnreachable: Bool = false, locationProblem: LocationProblem? = nil, hasSeenRegionOnboarding: Bool = false, hasSeenLocationPermissions: Bool = false, hasCameraProblem: Bool = false) {
```

Comment:
Very long line. Hard to read.  I prefer to have each parameter on a line, and any parameters (if any) that are worthy of special attention pointed out in comments.

Reply:
Yes we've discussed this already. This object is about to be simplified so it won't be quite so absurd.

Reply:
(also I have very little object permanence when it comes to this codebase.  I mostly recognise only what I see in front of me at the moment, if it connects to or even if something I've seen a few days ago I will just not notice :-).

Reply:
Ok, but if you keep making the same comments and I keep having to respond to explain why it's not an issue, that's a bit frustrating for me. So maybe you could stop?